src/CrudPanel.php

```diff
@@ -152,6 +154,24 @@ public function getRoute()
     }
 
     /**
+     * Get the breadcrumb for this item
+     *
+     * @return [Eloquent Collection]
+     **/
+    public function getBreadcrumb()
+    {
+        return $this->breadcrumb;
+    }
+
+    /**
+     * Set the breadcrumb for this route.
+     */
+    public function setBreadcrumb()
```

Comment:
And we probably could put a `$breadcrumb` in here just in case they want to set it to `something else`?

Or it could get renamed to `buildDefaultBreadcrumb` or something?

`setFOO` implies that one can set it to something one wants (but that there may be a default)?